src/librustdoc/html/render.rs

```diff
@@ -1352,6 +1352,7 @@ fn write_minify_replacer<W: Write>(
                         .into();
                     tokens.apply(|f| {
                         // We add a backline after the newly created variables.
+
```

Comment:
Stray change?

Reply:
(ping on this diff, I think this is accidental?)

Reply:
I don't understand where the difference comes from. I didn't change the file, and there is no log showing I changed it:
https://github.com/BaoshanPang/rust/commits/vxworks/src/librustdoc/html/render.rs